src/sentry/snuba/metrics/fields/base.py

```diff
@@ -1250,53 +1259,55 @@ def generate_where_statements(
             metric_mri=SessionMRI.ALL.value,
             metrics=[SessionMRI.SESSION.value],
             unit="sessions",
-            snql=lambda org_id, metric_ids, alias=None: all_sessions(
+            snql=lambda project_ids, org_id, metric_ids, alias=None: all_sessions(
                 org_id, metric_ids, alias=alias
             ),
         ),
         SingularEntityDerivedMetric(
             metric_mri=SessionMRI.ALL_USER.value,
             metrics=[SessionMRI.USER.value],
             unit="users",
-            snql=lambda org_id, metric_ids, alias=None: all_users(org_id, metric_ids, alias=alias),
+            snql=lambda project_ids, org_id, metric_ids, alias=None: all_users(
+                org_id, metric_ids, alias=alias
+            ),
         ),
         SingularEntityDerivedMetric(
             metric_mri=SessionMRI.ABNORMAL.value,
             metrics=[SessionMRI.SESSION.value],
             unit="sessions",
-            snql=lambda org_id, metric_ids, alias=None: abnormal_sessions(
+            snql=lambda project_ids, org_id, metric_ids, alias=None: abnormal_sessions(
                 org_id, metric_ids, alias=alias
             ),
         ),
         SingularEntityDerivedMetric(
             metric_mri=SessionMRI.ABNORMAL_USER.value,
             metrics=[SessionMRI.USER.value],
             unit="users",
-            snql=lambda org_id, metric_ids, alias=None: abnormal_users(
+            snql=lambda project_ids, org_id, metric_ids, alias=None: abnormal_users(
```

Comment:
Not necessarily a comment for this PR but as follow up, you no longer need to send down `org_id` because you can just call 
``` org_id_from_projects(projects) ````

Reply:
I think that passing the `org_id` is fine, so that we avoid using the `org_id_from_projects(projects)` function everywhere, especially considering that most `snql` functions use the `org_id`.